pkgs/tools/misc/remind/default.nix

```diff
@@ -1,6 +1,11 @@
-{lib, stdenv, fetchurl, tk, tcllib, makeWrapper
+{ lib
+, stdenv
+, fetchurl
+, tk
+, tcllib
+, makeWrapper
 , tkremind ? true
-} :
+}:
 
 assert tkremind -> tk != null;
 assert tkremind -> tcllib != null;
```

Comment:
This is not required and does not do anything.

Reply:
Unless someone uses overrides.

Reply:
You shouldn't just override any random input to null. New packages and updates should use boolean flags instead.